php/sunrise-router-annotations/composer.json

```
@@ -1,7 +1,8 @@
 {
   "require": {
     "sunrise/http-router": "~2.6.0",
-    "sunrise/http-factory": "*"
+    "sunrise/http-factory": "*",
+    "symfony/cache": "^5.0"
```

I find mixing `~` and `^` confusion, and not very accurate in this case
```suggestion
    "symfony/cache": "*"
```